src/Microsoft.HttpRepl/Commands/HelpCommand.cs

```diff
@@ -186,6 +186,7 @@ public static void CoreGetHelp(IShellState shellState, ICommandDispatcher<HttpSt
             output.AppendLine();
             output.AppendLine($"{"connect",navCommandColumn}{GetCommand<ConnectCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
             output.AppendLine($"{"set header",navCommandColumn}{GetCommand<SetHeaderCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"set queryString",navCommandColumn}{GetCommand<SetQueryStringCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
```

Comment:
`query-string`

Reply:
I definitely want to pull in @bradygaster on this one. I agree with the logic of what you said - including the query-param part... but I do wonder a bit about user expectations. Does it matter if (and this is just a guess on my part) our users would be more familiar with the term "query string" being used colloquially to cover a single query parameter? It seems like `query`/`query-string` would be aliases for each other (if we add those commands) and would be things the user might try on their own, but `query-param` might not be as obvious. I'd be happy to be wrong there, though, as I certainly prefer the precise language suggested here.

Reply:
Folks, during a recent GitHub service hiccup/incident, my original comment got deleted. The support staff at GitHub were kind enough to to recover the comment body, which I posting back here for context and belong right before @tlmii reply above:

> Why not make it simpler and call it `query`? Does the string part add or help anything except more typing? It would align nicely with [Section 3, “Syntax Components”](https://tools.ietf.org/html/rfc3986#section-3) (as well as [sub-section 3.4](https://tools.ietf.org/html/rfc3986#section-3.4)) of [RFC 3986](https://tools.ietf.org/html/rfc3986#section-3) where it's called just _query_:
> 
>     foo://example.com:8042/over/there?name=ferret#nose
>     \_/ \______________/\_________/ \_________/ \__/
>     | | | | |
>     scheme authority path query fragment
>     | _____________________|__
>     / \ / \
>     urn:example:animal:ferret:nose
> 
> If anything, `query-string` should be an alias like `dir` is for `ls`.
> 
> That said, there is another problem. The query component is opaque. The fact that it contains name-value pairs came from [URL-encoding of HTML forms](https://www.w3.org/TR/html401/interact/forms.html#h-17.13.3.4) and got adopted more widely outside the context of HTML form submission. The name-value pairs have become more conventionally known as _parameters_, e.g. [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get#Examples) and [query parameters in Open API](https://swagger.io/docs/specification/describing-parameters/#query-parameters), so I would argue that this should technically be called `query-param`. This way, `query`/`query-string` could be reserved to parsing and setting parameters given an entire “query string”. In other words, doing `set query foo=1&bar=2` would be equivalent to doing `set query-param foo 1` and `set query-param bar 2`.

Reply:
@ygalbel In the interest of moving this along (and since we won't push out a non-preview 5.1 release with this included for a while anyway) I'm going to go ahead and say we should go with what @atifaziz said: Make the command be `set query-param`. For now, lets leave it at that, without any additional `set query` or `set query-string` or aliases for any of it. We can see what the feedback is for the previews and get some more input from @bradygaster and others.